File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// WebsiteAnalysisApis SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class WebsiteAnalysisApisMakeContext
{
    public static function call(array $ctxmap, ?WebsiteAnalysisApisContext $basectx): WebsiteAnalysisApisContext
    {
        return new WebsiteAnalysisApisContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# WebsiteAnalysisApis SDK utility: make_context
require_relative '../core/context'
module WebsiteAnalysisApisUtilities
  MakeContext = ->(ctxmap, basectx) {
    WebsiteAnalysisApisContext.new(ctxmap, basectx)
  }
end
